grafana-plugin/integration-tests/auth.setup.ts

```diff
@@ -0,0 +1,69 @@
+import { BrowserContext, chromium, test as setup, expect } from '@playwright/test';
+import {
+  BASE_URL,
+  GRAFANA_PASSWORD,
+  GRAFANA_USERNAME,
+  GRAFANA_VIEWER_EMAIL,
+  GRAFANA_VIEWER_PASSWORD,
+  GRAFANA_VIEWER_USERNAME,
+} from './utils/constants';
+import config from '../playwright.config';
+import { configureOnCallPlugin } from './configurePlugin.setup';
+
+export const ADMIN_FILE = './adminState.json';
+export const VIEWER_FILE = './viewerState.json';
+
+setup('authenticate as admin', async ({ page }) => {
+  const { headless } = config.projects[0]!.use;
+  const browser = await chromium.launch({ headless, slowMo: headless ? 0 : 100 });
+  const browserContext = await browser.newContext();
+
+  const res = await login(browserContext, GRAFANA_USERNAME, GRAFANA_PASSWORD);
+
+  expect(res.ok()).toBeTruthy();
+
+  await page.context().storageState({ path: ADMIN_FILE });
+  await browserContext.storageState({ path: ADMIN_FILE });
```

Comment:
do we need both `storageState` calls here? see [here](https://github.com/grafana/oncall/pull/1649/files#diff-698159ea62969664ec910a06dd9c79739dcc39baacd386b29b299888a7e20569L55) in the previous `globalSetup.ts`.

Reply:
Yea, I was wondering the same, but in a few tests I tried skipping the `browserContext.storageState` call and it didn't work 🤔 I'll check for the other one to see if it is required, the first line comes from playwright documentation

Reply:
Oh good catch. I bet we won't be needing `storageState.json` any more, cause right now we should use auth instead.